rudder-agent/SOURCES/Makefile

```diff
@@ -628,15 +628,13 @@ build-cfengine: $(CFENGINE_DEPS) cfengine-source build-cfengine-stamp
 	if [ "$(CFENGINE_DEPS)" != "" ]; then ../../build-caching put dependencies/ $(DEPS_CACHE_PARAMETERS); fi
 
 build-cfengine-stamp:
-	# If there is no configure, bootstrap with autogen.sh first
-	cd cfengine-source && [ -x ./configure ] || NO_CONFIGURE=1 ./autogen.sh
```

Comment:
The configure is modified after this, so i think you should just run NO_CONFIGURE=1 ./autogen.sh unconditionnaly and do nor change the ./configure